include/ozo/detail/ostream.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <vector>
+#include <ostream>
+
+namespace ozo::detail {
+
+using std::ostream;
```

Comment:
Why std::ostream or what?

Reply:
But why we can't add it in the future? Interface should be the same with std::ostream.

Reply:
Sure, we can. I just did the same thing as for istream.h. The idea was to reserve place for custom stream implementation if it will be needed.

Reply:
If you want to do this to restrict clients from use of std::ostream in their code, then better idea is to make adaptor type. Because now code with std::ostream is still correct for compiler, therefore when you change this type later, clients could suffer.

Reply:
**Done**